src/extensions/core/load3d/Load3dAnimation.ts

```diff
@@ -10,164 +13,47 @@ class Load3dAnimation extends Load3d {
   isAnimationPlaying: boolean = false
 
   animationSpeed: number = 1.0
-  playPauseContainer: HTMLDivElement = {} as HTMLDivElement
-  animationSelect: HTMLSelectElement = {} as HTMLSelectElement
-  speedSelect: HTMLSelectElement = {} as HTMLSelectElement
 
   constructor(container: Element | HTMLElement) {
     super(container)
-    this.createPlayPauseButton(container)
-    this.createAnimationList(container)
-    this.createSpeedSelect(container)
   }
 
-  createAnimationList(container: Element | HTMLElement) {
-    this.animationSelect = document.createElement('select')
-    Object.assign(this.animationSelect.style, {
-      position: 'absolute',
-      top: '3px',
-      left: '50%',
-      transform: 'translateX(15px)',
-      width: '90px',
-      height: '20px',
-      backgroundColor: 'rgba(0, 0, 0, 0.3)',
-      color: 'white',
-      border: 'none',
-      borderRadius: '4px',
-      fontSize: '12px',
-      padding: '0 8px',
-      cursor: 'pointer',
-      display: 'none',
-      outline: 'none'
-    })
-
-    this.animationSelect.addEventListener('mouseenter', () => {
-      this.animationSelect.style.backgroundColor = 'rgba(0, 0, 0, 0.5)'
-    })
-
-    this.animationSelect.addEventListener('mouseleave', () => {
-      this.animationSelect.style.backgroundColor = 'rgba(0, 0, 0, 0.3)'
-    })
-
-    this.animationSelect.addEventListener('change', (event) => {
-      const select = event.target as HTMLSelectElement
-      this.updateSelectedAnimation(select.selectedIndex)
-    })
-
-    container.appendChild(this.animationSelect)
+  protected mountControls() {
+    const controlsMount = document.createElement('div')
+    controlsMount.style.pointerEvents = 'auto'
+    this.controlsContainer.appendChild(controlsMount)
+
+    this.controlsApp = createApp(Load3DAnimationControls, {
```

Comment:
Are you sure we need to create a separate Vue app here? Mounting a new app will lost all configurations such as themes  sync in the main app.

Reply:
I think I have to mount a new app to build UI on the top of threejs scence, which the only way I could figure it out...
currently, I don't use the configurations from main app in this UI yet.

Reply:
Is that because the container is mounted directly under document? If you mount the container element under #vue-app, you should be able to mount the component under the main App instance.

Reply:
each threejs container is under its own node's "image" filed Widget's HTMLElement
```
const sceneWidget = node.widgets.find((w: IWidget) => w.name === 'image')
const container = sceneWidget.element
```
and then
```
const rendererDomElement: HTMLCanvasElement = this.renderer.domElement
container.appendChild(rendererDomElement)
```

Reply:
Actually I understand your concern, but looks like it requires refactoring almost the whole code of load3d node

Reply:
Ah I see the issue is here: https://github.com/Comfy-Org/ComfyUI_frontend/blob/3d59d478b6ddab42251eba51af39e171d7b1e22a/src/scripts/domWidget.ts#L296-L298

All DOM widgets are actually outside of #vue-app, so you can not directly mount vue component in dom widget. We need to fix this issue definitely.

Reply:
I am going to merge this PR first as to get DOM widgets under #vue-app actually requires good amount of refactoring.